src/Tickets/Commerce/Gateways/Stripe/Webhooks/Handler.php

```diff
@@ -129,6 +129,27 @@ public static function get_handler_method_for_event( $type ) {
 	 * @return bool|WP_Error|null
 	 */
 	public static function update_order_status( \WP_Post $order, Commerce_Status\Status_Interface $status, array $metadata = [] ) {
+		if ( ! tribe( Order::class )->is_checkout_completed( $order->ID ) ) {
+
+			add_post_meta(
+				$order->ID,
+				'_tec_tickets_commerce_stripe_webhook_pending',
+				[
+					'new_status' => $status->get_wp_slug(),
+					'metadata'   => $metadata,
+					'old_status' => $order->post_status,
+				]
+			);
```

Comment:
If this is something we will add manually like this, create a method to enforce the array integrity.